practices/src/test/java/com/kingland/practice/buffer/LinkedBlockingQueueBufferTest.java

```diff
@@ -0,0 +1,88 @@
+package com.kingland.practice.buffer;
+
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class LinkedBlockingQueueBufferTest {
+
+    private static LinkedBlockingQueueBuffer buffer;
+
+    @BeforeEach
+    public void setup() {
+        buffer = new LinkedBlockingQueueBuffer(5);
+    }
+
+    @Nested
+    @DisplayName("shouldAddElementToBuffer")
+    class ShouldAddElementToBuffer {
+        @Test
+        @DisplayName("correctly")
+        void shouldAddElementToBufferCorrectly() {
+            // given
```

Comment:
PR2: Based on BDD, please provide some brief introduction from business perspective about what's `given`. Same comment for `when` and `then`.